phenopacketlab-restapi/src/main/resources/application-development.yml

```diff
@@ -21,12 +21,20 @@ phenopacketlab:
     - 'HP'
 
 api:
-  version: /api/v1
+  version: /api
```

Comment:
shouldn't we leave v1 here? as it relates to the version and not the path as used in the environment files?